client/src/app/plugins/PluginsRoot.js

```diff
@@ -88,6 +88,7 @@ export default class PluginsRoot extends PureComponent {
             subscribe={ subscribe }
             log={ this.log }
             displayNotification={ this.displayNotification }
+            _getGlobal={ app.getGlobal }
```

Comment:
Underscore as the first char suggests that it's a private property/method, while we pass it as part of the API here.

Still, I am not sure whether we really want to expose globals in the next release. How about if we hide it behind the feature flag too?

Reply:
The underscore was @nikku's idea after the meeting we did on how to approach this. We want to pass this for now, until we find a better solution about how to approach this issue. 

Maybe I should just pass the plugins instead of passing the globals?

Reply:
If it's a temporary solution then I don't bother. Please remember that the built in plugins will be included in this.

Alternatively, we could use `window.getAppPreload().plugins`, but I am not sure if it's a better solution ;).